content/en/history/d-machines/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: D-Machines*
```

Comment:
Is the intent to have all these placeholder document visible?  That seems like a lot of blank pages out the gate.  Maybe instead build out the frame work and mark the ones that need to be written as **draft**

```
draft: true
```